docs/ref/models/querysets.txt

```diff
@@ -813,10 +813,16 @@ duplicate values, use the ``all=True`` argument.
 of the type of the first ``QuerySet`` even if the arguments are ``QuerySet``\s
 of other models. Passing different models works as long as the ``SELECT`` list
 is the same in all ``QuerySet``\s (at least the types, the names don't matter
-as long as the types in the same order).
+as long as the types in the same order). In such cases, you must use column
+names from the first ``QuerySet`` in ``order_by()``. For example::
```

Comment:
"in QuerySet methods applied to the resulting QuerySet"?
(besides order_by(), it also applies to values/values_list)